README.md

```diff
@@ -20,8 +20,9 @@ see cmake options for details.
   - [Contents of the repo](#contents-of-the-repo)
   - [Integration](#integration)
     - [Weekly tags](#weekly-tags)
-  - [Third-Party tools](#third-party-tools)
-  - [Building](#building)
+3. [Third-Party tools](#third-party-tools)
```

Comment:
Perhaps something went wrong with (previous?) merge/cherry-pick, but this TOC looks lolzy now 😉